index.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: default
+title: home
+---
+### A weekly review club for Bitcoin Core PRs
+
+<span class="question">What is this?</span> A weekly
+club for reviewing Bitcoin Core PRs in the Freenode `#bitcoin-core-pr-reviews`
```

Comment:
Perhaps "IRC" here, and the more detailed "Freenode" in the How section.

Reply:
done